llms/hf_llm/convert.py

```diff
@@ -33,6 +33,22 @@ def fetch_from_hub(hf_path: str):
     )
     return weights, config.to_dict(), tokenizer
 
+def fetch_from_local(local_path):
+    weight_files = glob.glob(f"{local_path}/*.safetensors") + glob.glob(f"{local_path}/*.bin")
+    if len(weight_files) == 0:
+        raise FileNotFoundError("No safetensors or bin files found in {}".format(local_path))
```

Comment:
What is `*.bin`?

Reply:
The .bin files are regular pytorch files Like the ones you would find in on HF that don't use safetensors. For example, WizardLM/WizardMath-7B-V1.1

Reply:
Makes sense, thanks. We do not support PyTorch format, so I would remove that part of the code.

Reply:
Sure thing, I removed it.